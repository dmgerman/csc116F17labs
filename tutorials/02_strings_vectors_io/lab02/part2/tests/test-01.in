This is the Sea.
And this is not.

The line above is empty
This is the Sea.
Another line
This is the Sea.
Another line
