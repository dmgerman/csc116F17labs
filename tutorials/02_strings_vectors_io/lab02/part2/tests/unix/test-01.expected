Another line
The line above is empty

And this is not.
This is the Sea.
